pkg/sso/README.md

```diff
@@ -16,19 +16,19 @@ import (
 )
 
 func main() {
-    sso.SetAPIKey("my_api_key")
+    sso.Configure(
+        "xxxxx",                            // WorkOS api key
+        "project_xxxxx",                    // WorkOS project id
+        "https://mydomain.com/callback",    // Redirect URI
+    )
 
     http.Handle("/login", sso.Login(sso.GetAuthorizationURLOptions{
```

Comment:
```suggestion
    http.Handle("/login", sso.Login(sso.GetAuthorizationURLOptions {
```

Maybe run this code thru the formater?

Reply:
Formatter does not work in the readme. But my bad, I have been lazy.

Reply:
Actually I just double checked it on vscode, seems alright

```go
sso.GetAuthorizationURLOptions{
```
no space on this one after options